src/seedsigner/helpers/embit_utils.py

```diff
@@ -93,3 +98,26 @@ def get_multisig_address(descriptor: Descriptor, index: int = 0, is_change: bool
     elif descriptor.is_taproot:
         # TODO: Not yet implemented!
         raise Exception("Taproot verification not yet implemented!")
+
+
+
+def sign_message(seed_bytes: bytes, derivation, msg: bytes, compressed: bool = True, embit_network: str="main") -> bytes:
+    """
+        from: https://github.com/cryptoadvance/specter-diy/blob/b58a819ef09b2bca880a82c7e122618944355118/src/apps/signmessage/signmessage.py
+    """
+    """Sign message with private key"""
+    msghash = sha256(
+        sha256(
+            b"\x18Bitcoin Signed Message:\n" + compact.to_bytes(len(msg)) + msg
+        ).digest()
+    ).digest()
+
+    root = bip32.HDKey.from_seed(seed_bytes, version=NETWORKS[embit_network]["xprv"])
+    prv = root.derive(derivation).key
+    sig = secp256k1.ecdsa_sign_recoverable(msghash, prv._secret)
+    flag = sig[64]
+    sig = ec.Signature(sig[:64])
+    c = 4 if compressed else 0
```

Comment:
any plans to support https://github.com/bitcoin/bips/blob/master/bip-0137.mediawiki?

Reply:
here is also draft of new thing (hard to say if it gets formalized) but it is basically the same as BIP137 (if fixed)

check this PR where I provided easy to use table https://github.com/ZenulAbidin/bips/pull/3/files

Reply:
For now, my goal is to keep this as an exact copy-and-paste from the Specter DIY implementation in order to guarantee compatibility with Specter Desktop.

Future revisions would probably make sense as PRs into `embit` in order to maintain that same ecosystem-wide (SeedSigner, Specter DIY, Specter Desktop) compatibility.